src/odemis/gui/comp/miccanvas.py

```diff
@@ -1820,11 +1820,11 @@ def zoom_out(self):
         """
         logging.debug("Zooming out to show all scintillators.")
         if self._tab_data_model:
-            sz = self._tab_data_model.scintillator_size
-            l = min(list(self._tab_data_model.scintillator_positions.values()), key=lambda item: item[0])[0] - sz[0]
-            t = max(list(self._tab_data_model.scintillator_positions.values()), key=lambda item: item[1])[1] + sz[1]
-            r = max(list(self._tab_data_model.scintillator_positions.values()), key=lambda item: item[0])[0] + sz[0]
-            b = min(list(self._tab_data_model.scintillator_positions.values()), key=lambda item: item[1])[1] - sz[1]
+            sz = self._tab_data_model.main.scintillator_size
+            l = min(list(self._tab_data_model.main.scintillator_positions.values()), key=lambda item: item[0])[0] - sz[0]
+            t = max(list(self._tab_data_model.main.scintillator_positions.values()), key=lambda item: item[1])[1] + sz[1]
+            r = max(list(self._tab_data_model.main.scintillator_positions.values()), key=lambda item: item[0])[0] + sz[0]
+            b = min(list(self._tab_data_model.main.scintillator_positions.values()), key=lambda item: item[1])[1] - sz[1]
```

Comment:
Shouldn't we just use stage.POS_ACTIVE_RANGE? I feel it's exactly what it's supposed to represent.

Reply:
We could in principle do that. There is still the question, what position POS_ACTIVE_RANGE should be. I think you originally wanted it to indicate the corner scintillator positions. Now I use it as the corner sample carrier position (scintillator + border) because that made the background rectangle coordinates a bit easier. Still, when zooming out, we don't want to show the sample carrier border, only the scintillators. So the question is, should we change the meaning of POS_ACTIVE_RANGE, update the backgorund coordinates (border has negative values) and make the change here? Not sure if it's worth the effort to be honest. On the other hand, using the scintillator positions in POS_ACTIVE_RANGE might be more intuitive and easier to understand? What do you think?

Reply:
Right, indeed, I forgot about the extra borders that POS_ACTIVE_RANGE include.

I would note that this metadata is supposed to represent the maximum area the user can go with the stage. Soon the GUI will automatically prevent the user from going outside of it. So maybe it'd make sense to have the range limited to the scintillator's area anyway. If that's the case, then you could use it here. For now, let's leave it as-is.
Could you check with Job what's the maximum area we want the user to be able to move to?

Reply:
The stage moves are limited by the "range" parameter in the "axes" argument of the piezomotor driver. By default, it's set to the scintillator area, so the movements are already limited to that are in the GUI.